### PR TITLE
helper for website ID <-> domain/slug

### PR DESCRIPTION
**Pull request by Chive**
Allows traversal through the control panel api from website ``domain / slug`` to ``id`` and vice versa.

Also includes an update in the project list view + adds a pager there.

### CODE REVIEW

File: aldryn_client/cli.py [hunk 1]
```diff
@@ -71,12 +71,13 @@ def project_list(obj):
...
     projects = [(
         project_data['id'],
+        project_data['domain'],
         project_data['name'],
         organisations.get(project_data['organisation_id'], 'Personal'),
     ) for project_data in data['websites']]
 
-    header = ['ID', 'Name', 'Organisation']
-    click.echo(table(projects, header))
+    header = ['#', 'ID', 'Name', 'Organisation']
```
Comment (GaretJax) [inline aldryn_client/cli.py]: So the domain is the new ID?

File: aldryn_client/cloud.py [hunk 2]
```diff
@@ -85,6 +85,20 @@ def upload_boilerplate(self, archive_obj):
         )
         return request.request()
 
+    def get_website_id_for_slug(self, slug):
+        request = api_requests.SlugToIDRequest(
+            self.session,
+            url_kwargs={'website_slug': slug}
+        )
+        return request.request()
```
Comment (GaretJax) [inline aldryn_client/cloud.py]: What about adding `__call__` to `APIRequest`?

**Comment by GaretJax** [comment]
Left a comment, the column should probably be renamed, but up to you. LGTM otherwise.

File: aldryn_client/cli.py (hunk 1 above)
Comment (Chive) [inline aldryn_client/cli.py]: Yeah. I think this is the best approach.

Comment (GaretJax) [inline aldryn_client/cli.py]: Is it the full domain (with aldryn.io) or just the subdomain (e.g. the slug)?

**Comment by GaretJax** [comment]
LGTM

**Pull request merged**